Assets/Plugins/LeapMotion/Modules/Hands/Scripts/HandBinder/HandBinderAutoBinder.cs

```diff
@@ -49,12 +43,15 @@ public static class HandBinderAutoRigger {
             handBinder.boundHand.fingers[4].boundBones = AssignUnityBone(pinkyBones);
```

Comment:
What's the AssignUnityBone call about?

Reply:
It converts Transforms that were found in the scene into a serializable data structure that I have defined in the HandBinderData.cs.